service/k8sclient/k8sclient_test.go

```diff
@@ -1014,7 +1107,37 @@ func TestGetClusterState(t *testing.T) {
 			crAndPodsVersionMatches: true,
 			expectedState:           ClusterStateChanging,
 		},
+	}
+	ctx := context.Background()
+	kubeconfig, err := ioutil.ReadFile(os.Getenv("HOME") + "/.kube/config")
```

Comment:
interesting case: in some cases we use kube config from `/.kube/config`, and in some cases we use the one from `PERCONA_TEST_DBAAS_KUBECONFIG` environment variable.

Reply:
ci has
```
 echo "${BASE64_KUBECONFIG}" > "${PATH_TO_KUBECONFIG}"
```
on setupenv step for EKS.